client/status/v2.go

```diff
@@ -51,17 +50,18 @@ type StatusV2 interface {
 // Session failures:
 //   - [session.StatusTokenNotFound]: *[SessionTokenNotFound];
 //   - [session.StatusTokenExpired]: *[SessionTokenExpired];
-func FromStatusV2(st *status.Status) Status {
+func FromStatusV2(st *status.Status) error {
```

Comment:
can we rename it a little? we have a common pattern ("ReadFromV2", "WriteToV2") so it was obvious before (the v2 type to the sdk type) but not that obvious now (at least add "Err"?)

the same about `ToStatusV2`

cc @roman-khimov, @cthulhu-rider

Reply:
Is it ok?
```
FromStatusV2 -> ReadErrorFromV2
ToStatusV2 -> WriteErrorToV2
```

Reply:
ErrorFromV2, ErrorToV2, maybe. From/To already is enough, Read/Write just duplicates it. I'm OK with the change since now `apistatus` is all about errors and doesn't have any mysterious `Status` type.

Reply:
This variant is better, renamed to it